modules/vantagecloud-lake/pages/vantagecloud-lake-compute-cluster-airflow.adoc

```diff
@@ -0,0 +1,341 @@
++
+--
+[source, bash]
+----
+AIRFLOW_VERSION=2.9.2
+PYTHON_VERSION="$(python3 --version | cut -d " " -f 2 | cut -d "." -f 1-2)"
+CONSTRAINT_URL="https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION}.txt"
+pip install "apache-airflow==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"
+----
+--
+Linux::
++
+--
+[source, bash]
+----
+AIRFLOW_VERSION=2.9.2
+PYTHON_VERSION="$(python3 --version | cut -d " " -f 2 | cut -d "." -f 1-2)"
+CONSTRAINT_URL="https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION}.txt"
+pip install "apache-airflow==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"
+----
+--
+====
+
+3. Install the Airflow Teradata provider stable version from PyPI repository.
++
+[source, bash]
+----
+pip install "apache-airflow-providers-teradata"
+----
+
+== Install dbt
+1. Create a new python environment to manage dbt and its dependencies. Activate the environment:
++
+[tabs]
+====
+Windows::
++
+--
+[source, bash]
+----
+python -m venv env
+source env/bin/activate
+----
+--
+MacOS::
++
+--
+[source, bash]
+----
+python3 -m venv env
+source env/bin/activate
+----
+--
+Linux::
++
+--
+[source, bash]
+----
+python3 -m venv env
+source env/bin/activate
+----
+--
+====
+
+2. Install `dbt-teradata` and `dbt-core` modules:
++
+[source, bash]
+----
+pip install dbt-teradata dbt-core
+----
+
+== Create a database
+
+Let's create `jaffle_shop` database in Vantage lake instance with TD_OFSSTORAGE as default
+
+[source, teradata-sql]
+----
+CREATE DATABASE jaffle_shop
+AS DEFAULT STORAGE = TD_OFSSTORAGE OVERRIDE ON ERROR,
+PERMANENT = 120e6, -- 120MB
+    SPOOL = 120e6; -- 120MB
+----
+
+== Create a database user
+
+Let's create `lake_user` user in Vantage lake instance.
+
+[source, teradata-sql]
+----
+CREATE USER lake_user
+AS PERMANENT = 1000000,
+PASSWORD = lake_user,
+SPOOL = 1200000,
+DEFAULT DATABASE = jaffle_shop;
+----
+
+== Grant access to user
+
+Let's provide required privileges to user `lake_user` to manage compute cluster.
+
+[source, teradata-sql]
+----
+GRANT CREATE COMPUTE GROUP To lake_user;
+GRANT DROP COMPUTE GROUP TO lake_user;
+GRANT SELECT ON DBC TO lake_user;
+GRANT ALL ON jaffle_shop TO lake_user;
+----
+
+== Setup dbt project
+
+1. Clone the jaffle_shop repository and cd into the project directory:
++
+[source, bash]
+----
+git clone https://github.com/Teradata/jaffle_shop-dev.git jaffle_shop
+cd jaffle_shop
+----
+2. Create `profiles.yml` file. Add the following config to `profile.yml` file. Adjust `<host>` to match your Teradata Vantage lake instance.
++
+[source, yaml]
+----
+jaffle_shop:
+  outputs:
+    dev:
+      type: teradata
+      host: <host>
+      user: lake_user
+      password: lake_user
+      logmech: TD2
+      schema: jaffle_shop
+      tmode: ANSI
+      threads: 1
+      timeout_seconds: 300
+      priority: interactive
+      retries: 1
+  target: dev
+----
+
+== Define DAG in Airflow
```

Comment:
@sc250072 Maybe we can add a bit more details on how to name the file and where to store it.

Reply:
Yes. Added with details on how and where to save